client/src/views/account/AccountSettingsPage.vue

```diff
@@ -0,0 +1,208 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div class="account-settings-page-container">
+    <ion-radio-group class="tabs-menu">
+      <div class="menu-content">
+        <div class="menu-list">
+          <!-- Settings -->
+          <ion-radio
+            :value="AccountSettingsTabs.Settings"
+            :class="{ 'radio-checked': activeTab === AccountSettingsTabs.Settings }"
+            class="menu-list-item"
+            @click="onTabChange(AccountSettingsTabs.Settings)"
+          >
+            <ion-text class="menu-list-item__title button-medium">
+              {{ $msTranslate('HomePage.tabs.settings') }}
+            </ion-text>
+          </ion-radio>
+          <!-- Account -->
+          <ion-radio
+            :value="AccountSettingsTabs.Account"
+            :class="{ 'radio-checked': activeTab === AccountSettingsTabs.Account }"
+            class="menu-list-item"
+            @click="onTabChange(AccountSettingsTabs.Account)"
+          >
+            <ion-text class="menu-list-item__title button-medium">
+              {{ $msTranslate('HomePage.tabs.account') }}
+            </ion-text>
+          </ion-radio>
+          <!-- Authentication -->
+          <ion-radio
+            :value="AccountSettingsTabs.Authentication"
+            :class="{ 'radio-checked': activeTab === AccountSettingsTabs.Authentication }"
+            class="menu-list-item"
+            @click="onTabChange(AccountSettingsTabs.Authentication)"
+          >
+            <ion-text class="menu-list-item__title button-medium">
+              {{ $msTranslate('HomePage.tabs.authentication') }}
+            </ion-text>
+          </ion-radio>
+        </div>
+      </div>
+    </ion-radio-group>
+    <div class="profile-content">
+      <!-- Settings tab -->
+      <div
+        v-if="activeTab === AccountSettingsTabs.Settings"
+        class="profile-content-item"
+      >
+        <div class="item-header">
+          <ion-text class="item-header__title title-h3">{{ $msTranslate('HomePage.profile.settings.title') }}</ion-text>
+          <ion-text class="item-header__description body">{{ $msTranslate('HomePage.profile.settings.description') }}</ion-text>
+        </div>
+        <settings-list />
+      </div>
+      <!-- account tab -->
+      <div
+        v-if="activeTab === AccountSettingsTabs.Account"
+        class="profile-content-item"
+      >
+        <div class="item-header">
+          <ion-text class="item-header__title title-h3">{{ $msTranslate('HomePage.profile.account.title') }}</ion-text>
```

Comment:
Missing a lot of translations here.

![Uploading Screenshot 2025-06-17 064515.png…]()